### ISSUE TITLE
[Error] Could not test Plex - Can't connect to Plex Server when set to Require Secure Connections

### ISSUE THREAD

**Issue by phaelon74** [tidusjar/Ombi#2909]
**Describe the bug**
Ombi is unable to connect to Plex when plex is set with Required for "Secure Connections".

**To Reproduce**
1. Install Ubuntu 16.04, Install Ombi via Package (Followed Install instructions from Ombi Wiki)
2. Set Plex Server to "Required" for Secure Connections  (it will fail)
3. Go to plex and set it to "Preferred" secure connections
4. Go back to Ombi, and tell it to use SSL.  (It will fail)
5. Tell Ombi to NOT use SSL (IT WILL WORK) {Plex is still set to Preferred}
5a. Run a packet capture on Plex box, you will see failed hand shakes and inability to choose cipher.  Final landing is unsecure channel. 

**Expected behavior**
Plex is set to Require secure connections and Ombi is set to SSL.  A secure, SSL handshake occurs between ombi and Plex.

**Screenshots**

**Logs (Logs directory where Ombi is located)**
2019-04-01 22:12:03.062 -04:00 [Error] Could not test Plex
System.Net.Http.HttpRequestException: Connection timed out ---> System.Net.Sockets.SocketException: Connection timed out
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Ombi.Api.OmbiHttpClient.SendAsync(HttpRequestMessage request) in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 61
   at Ombi.Api.Api.Request[T](Request request) in C:\projects\requestplex\src\Ombi.Api\Api.cs:line 38
   at Ombi.Api.Plex.PlexApi.GetStatus(String authToken, String uri) in C:\projects\requestplex\src\Ombi.Api.Plex\PlexApi.cs:line 97
   at Ombi.Controllers.External.TesterController.Plex(PlexServers settings) in C:\projects\requestplex\src\Ombi\Controllers\External\TesterController.cs:line 246

**Desktop (please complete the following information):**
 - Ubuntu 16.04

**Ombi Version (please complete the following information):**
 - Version: 3.0.4256
- Media Server: Plex - Version 1.15.2.793

**Additional context**
My guess is, Cipher mismatch regarding exchange of token.

**Comment by tidusjar** [comment]
This is sort of outside of Ombi, Since Ombi doesn't natively support SSL.

I'm no where near an expert in SSL/TLS cryptography. All Ombi does when you select the SSL checkbox is talk to Plex over HTTPS and not HTTP.

**Comment by RobinMM** [comment]
meaning this will never work for ombi? Having same issue. Who wont force secured connections now a days? For tautulli it works fine with ssl enabled.

**Comment by tidusjar** [comment]
Have you tried enabling the Ignore SSL Errors in the Ombi Settings?

**Comment by RobinMM** [comment]
I have tried that, and not working. Still same error occurs and because of this I cannot load libraries etc.

**Comment by RobinMM** [comment]
Update on this, it did work with setting: Ignore SSL errors. But had to restart service before it worked.

**Comment by phaelon74** [comment]
Correct, I also found that eventually as well.  IT was a different GitHub post that stated you need to restart after changing this setting.

@RobinMM if you have Plex set to ONLY allow secure connections, as in Require, then yes, it would have to connect via secure methods.

**Comment by t1hom7as** [comment]
Can confirm that I ran into this issue too, I resolved this by checking the SSL box in the Plex configuration in Ombi, then changing my Plex server from "required" to "preferred" and also checking the "Ignore any certificate errors" box in the Ombi settings. This got the connection working, I then changed the Plex server back to "required". 
I did also change my "hostname/ip" from the auto discovered IP (a private one) to my hostname.
